pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
Does `tess4j` include binaries for all the platforms that we support? (Windows, Mac and Linux). Does it require that Tesseract is present in the machine that Tabula is used?